src/modules/navigator/mission_block.cpp

```diff
@@ -211,6 +212,13 @@ MissionBlock::mission_item_to_position_setpoint(const struct mission_item_s *ite
 		sp->type = position_setpoint_s::SETPOINT_TYPE_LOITER;
 		break;
 
+	case NAV_CMD_RETURN_TO_LAUNCH:
+		sp->type = position_setpoint_s::SETPOINT_TYPE_RTL;
+		sp->lat = _navigator->get_home_position()->lat;
+		sp->lon = _navigator->get_home_position()->lon;
+		sp->alt = _navigator->get_global_position()->alt;
+		break;
+
 	default:
 		sp->type = position_setpoint_s::SETPOINT_TYPE_POSITION;
```

Comment:
The default case here still needs to be fixed. We just covered this one corner case, but we didn't fix the root bug, which is to *NOT* have a default statement like this one, but to skip the (unsupported) item and continue with the next supported mission item (or abort / end the mission - that's an architecture question).